include/pqxx/stream_to.hxx

```diff
@@ -470,22 +474,23 @@ private:
 
 template<typename Columns>
 inline stream_to::stream_to(
-  transaction_base &tx, std::string_view table_name, Columns const &columns) :
-        stream_to{tx, table_name, std::begin(columns), std::end(columns)}
+  transaction_base &tx, std::string_view table_name, Columns const &columns,
+  bool binary /*= false*/) :
```

Comment:
I like the _idea_ of adding that `/*= false*/` documentation, but I have a _practical_ problem with it: it creates a kind of "double bookkeeping" between the header and the source file, which we will have to keep consistent.  And if we ever get that wrong, nothing will warn us about it.  We'll just have a bit of misleading documentation in the source code.